triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -197,15 +200,22 @@ object Converter {
         QualifiedName(
           DottedName.assertFromString("DA.Internal.LF"),
           DottedName.assertFromString("AnyTemplate")))
+    val templateTy = Identifier(
+      PackageId.assertFromString(created.getTemplateId.packageId),
+      QualifiedName(
+        DottedName.assertFromString(created.getTemplateId.moduleName),
+        DottedName.assertFromString(created.getTemplateId.entityName))
+    )
```

Comment:
This pattern shows up in a couple of places, might make sense to factor it out.

Reply:
Yeah, I was planning to do this as part of factoring out the conversion logic from triggers and DAML script.